lib/api/survey_api_service.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:dio/dio.dart';
+import 'package:retrofit/http.dart';
+import 'package:survey_flutter/model/response/surveys_container_response.dart';
+
+part 'survey_api_service.g.dart';
+
+@RestApi()
+abstract class SurveyApiService {
+  factory SurveyApiService(Dio dio, {String baseUrl}) = _SurveyApiService;
+
+  @GET('/surveys?page[number]={page_number}&page[size]={page_size}')
+  Future<SurveysContainerResponse> getSurveys(
+    @Path('page_number') int pageNumber,
+    @Path('page_size') int pageSize,
+  );
```

Comment:
IMO, we can use `@Query` instead of `@Path` here.
```Dart
@GET('/surveys')
Future<SurveysContainerResponse> getSurveys(
  @Query('page[number]') int pageNumber,
  @Query('page[size]') int pageSize,
);
```